presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+* Introduced BigQuery Connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve ORC writer performance for string columns.
+* Add support for improving cache affinity hit rate by preventing repartitioning when nodes go down for a limited time. This can be enabled by setting the `internal-communication.memoize-dead-nodes-enabled` configuration property
```

Comment:
configuration properties go under double back quotes - 

```
``internal-communication....``
```

and generally we mention the whole thing - 

```
this can be enabled by setting the <x> configuration property to ``value``
```

Reply:
Fixed in latest revision